mlem/utils/templates.py

```diff
@@ -22,6 +22,7 @@ def generate(self, **additional):
         j2 = Environment(
             loader=FileSystemLoader(self.templates_dir + [self.TEMPLATE_DIR]),
             undefined=StrictUndefined,
+            autoescape=True,
```

Comment:
One more example, `test_dockerfile_generator_super_custom`. It has to do something with your changes, cause I just run CI in a nearby PR #443 and it worked for me. I assume it is this change. Maybe we need to fix the test (if everything works if you `docker build` that custom dockerfile - have to check it), maybe we need to disable the check here (if this is ok to not escape things in dockerfiles). Relying on you to find this out :)